client/js/src/client.ts

```diff
@@ -1053,6 +1050,17 @@ export function api_factory(
 						pending_stream_messages[event_id].push(_data);
 					}
 				};
+				event_stream.onerror = async function (event) {
```

Comment:
Not completely related, but wanted to have an error if the connection broke. Previously, if you closed the backend while a function was running, the frontend would indefintely show "processing". Now it clearly displays an error message.